tests/test_items.py

```diff
@@ -30,9 +32,14 @@
     [(PaginationLink, example_product_list_result["productList"]["paginationNext"])] +  # type: ignore
     [(ProductList, example_product_list_result["productList"])]  # type: ignore
 )  # type: ignore
-def test_item(cls, data):
-    item = cls.from_dict({**data, "unexpected_attribute": "Should not fail"})
+@pytest.mark.parametrize(
+    "unexpected_attrs",
+    [{}, {"unexpected_attribute": "Should not fail"}]
+)  # type: ignore
+def test_item(cls, data, unexpected_attrs):
+    item = cls.from_dict({**data, **unexpected_attrs})
```

Comment:
Would it make sense to add some `__slots__`-check test? I assume, any item has a built-in `_additional_attrs` slot now, and it should stay that way?

Reply:
There is already and `__slots__` check here: https://github.com/scrapinghub/autoextract-poet/blob/1c1921822952f4717b7924d7d427bbd7b87255c4/tests/test_items.py#L50-L52 

Do you think is not enough?